recipes/tkwant/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "tkwant" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://gitlab.kwant-project.org/kwant/{{ name }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
+  sha256: 21fdf5ed092ada62363d38ec352b07805b6795e2105400fbae1625675523ac06
+  patches:
+    - requirement_patch.diff
+
+build:
+  skip: true  # [py2k]
+  skip: true  # [py>=38]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - setuptools
+    - libblas
+    - libcblas
+    - cython
+    - python
+    - numpy
+    - scipy
+    - pytest-runner
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - scipy
+    - pytest
+    - sympy
+    - tinyarray
+    - mpmath
+    - matplotlib
```

Comment:
This should be `matplotlib-base`

Reply:
Dear @scopatz , I changed matplotlib to matplotlib-base, but there is an "error: command 'gcc' failed: No such file or directory: 'gcc'" (link below). It does not seem to be related to matplotlib, and when I turn back the change and test in a container on my local machine, the same error appears. Have you seen this problem elsewhere, it is mysterious to me why the linux pipeline is not working any more, as it did 3 days before?

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=203036&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0&l=741

Reply:
Yeah that seems unrelated to matplotlib-base, CC @conda-forge/staged-recipes

Reply:
@conda-forge/staged-recipes : Running the same recipe again today the problem disappeared.

Reply:
Ok! This looks good now